core/common/src/main/java/alluxio/metrics/MetricsSystem.java

```diff
@@ -470,24 +507,56 @@ public static Timer timer(String name) {
    * The synchronized keyword is added for correctness with {@link #resetAllMetrics}
    */
   private static synchronized List<alluxio.grpc.Metric> reportMetrics(InstanceType instanceType) {
```

Comment:
Only report the should report metrics (MetricKey.isClusterAggregated == true, predefined) instead report all the worker or client metrics.